projects/angular/src/emphasis/_labels.clarity.scss

```diff
@@ -157,6 +185,34 @@
     }
 
     &.clickable {
+      @include mixins.css-var(
```

Comment:
Anchor based clickable labels with badges have underline spilled to badges. this is not desirable situation.

![image](https://github.com/vmware-clarity/ng-clarity/assets/6692908/c1bc25f4-405e-489e-a9ec-1ab80e21931b)

Reply:
Created an issue in Jira for that.